File: TrocaDeTela/app/src/main/java/com/android/trocadetela/MainActivity.java
package com.android.trocadetela;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

	Button proximoButton;

	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		setContentView(R.layout.activity_main);

		proximoButton = (Button) findViewById(R.id.proximoButton);

		proximoButton.setOnClickListener(new View.OnClickListener() {
			@Override
			public void onClick(View view) {
				Intent intent = new Intent(MainActivity.this, SegundaActivity.class);
				startActivity(intent);
				finish();
			}
		});
	}
}

File: Kotlin/GasolinaouAlcool/app/src/main/java/br/com/teamcode/gasolinaoualcool/activity/MainActivity.kt
package br.com.teamcode.gasolinaoualcool.activity

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import br.com.livroandroid.carros.extensions.setupToolbar
import br.com.teamcode.gasolinaoualcool.R
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        setupToolbar(R.id.toolbar)

        calcularButton.setOnClickListener {
            val ok = alcoolEditText.text.isNotEmpty() && gasolinaEditText.text.isNotEmpty()

            if(ok) {
                val percentual = alcoolEditText.text.toString().toDouble() / gasolinaEditText.text.toString().toDouble()

                if(percentual <= 0.70) {
                    resultadoTextView.text = "Melhor usar álcool"
                } else {
                    resultadoTextView.text = "Melhor usar gasolina"
                }
            }
        }

    }
}
